prboom2/src/SDL/i_system.c

```diff
@@ -46,6 +46,7 @@
 #include <time.h>
 #include <signal.h>
 #include <string.h>
+#include <dirent.h>
```

Comment:
There is no dirent.h on Windows. Please use stat() and S_ISDIR instead, c.f. the code in d/main.c:IdentifyVersion().

Reply:
Woops. Latest commit should fix that.